tests/utils/pouchdb.js

```diff
@@ -1,3 +1,7 @@
 module.exports = require('pouchdb-core')
   .plugin(require('pouchdb-replication'))
   .plugin(require('pouchdb-adapter-memory'))
+
+// module.exports = require('pouchdb-monorepo/packages/node_modules/pouchdb-core')
+  // .plugin(require('pouchdb-monorepo/packages/node_modules/pouchdb-replication'))
+  // .plugin(require('pouchdb-monorepo/packages/node_modules/pouchdb-adapter-memory'))
```

Comment:
Yes, I was wondering about that, too.  I should have removed it. Sorry about that!